awx/main/tasks/host_metrics.py

```diff
@@ -0,0 +1,202 @@
+import datetime
+from dateutil.relativedelta import relativedelta
+import logging
+
+from django.conf import settings
+from django.db.models import Count
+from django.db.models.functions import TruncMonth
+from django.utils.timezone import now
+from awx.main.dispatch import get_task_queuename
+from awx.main.dispatch.publish import task
+from awx.main.models.inventory import HostMetric, HostMetricSummaryMonthly
+from awx.conf.license import get_license
+
+logger = logging.getLogger('awx.main.tasks.host_metric_summary_monthly')
+
+
+@task(queue=get_task_queuename)
+def host_metric_summary_monthly():
+    """Run cleanup host metrics summary monthly task each week"""
+    from awx.conf.models import Setting
```

Comment:
this will probably fail linters without a NOQA flag as well. If the model is needed, I expect you probably do need an in-line import to avoid circular imports. Before you do that, I would like to ask if the alternative would work to just reference `settings.HOST_METRIC_SUMMARY_TASK_LAST_TS`.